src/hooks/useSections/useActiveSections.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable max-params */
```

Comment:
It's not even needed here anymore I'm removing it.

I'd rather we leave it since it gives an indicator even we bypass it that the hook might need refacotoring or split.